scripts/install_bazel.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+set -e
+
+LOCAL_OS="$(uname)"
+
+if [ -z "$BAZEL_VERSION" ]; then
+   BAZEL_VERSION="0.11.1"
+fi
+
+if [ -z "$BAZEL_INSTALL" ]; then
+   BAZEL_INSTALL="${HOME}/bazel/install"
+fi
+
+mkdir -p $BAZEL_INSTALL
+pushd $BAZEL_INSTALL
+
+if [ "$LOCAL_OS" == "Linux" ]; then
```

Comment:
Does it make sense to check specifically for Debian based Linux, as the we are using Debian package installer here?

Reply:
That's an interesting point, while this is currently strictly for CI which we define as `trusty` in `.travis.yml`. Perhaps renaming the file to be a bootstrap-travis or something would make more sense for now and as that requirement changes we could iterate.

Reply:
I can swap the check to be `TRAVIS_OS_VERSION` and ensure we are in `CI` to prevent unintended use-cases.